yarn.lock

```diff
@@ -0,0 +1,301 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Should we be ignoring this?

Reply:
According to Buchis reference PR no...
https://github.com/4teamwork/ftw.slider/pull/94/files

Reply:
Good logic :-) I'll wait to see what you do :-)

Reply:
I'll leave it there for now.